src/composables/useNumbers.ts

```diff
@@ -164,6 +166,10 @@ export default function useNumbers() {
       number = 0;
     }
 
+    if (options.returnLessBasicPoint && number > 0 && number < 0.00005) {
+      return '< 0.01%';
```

Comment:
I think we should prob be consistent and always return it without passing a config option.
Thoughts @garethfuller?